docs/frameworks/symfony.md

```diff
@@ -138,59 +138,48 @@ in `trusted_proxies`.
 
 ## Assets
 
-To deploy Symfony websites, assets needs to be served by AWS S3. Setting up the S3 bucket is already explained in
-the [websites documentation](../websites.md#hosting-static-files-with-s3). This section provides additional instructions specific to Symfony assets and Webpack
-Encore.
+To deploy Symfony websites, assets needs to be served from AWS S3. The easiest solution to do this is to use the
+[Server-side website construct of the Lift plugin](https://github.com/getlift/lift/blob/master/docs/server-side-website.md).
 
-First, you need to [tell Symfony](https://symfony.com/doc/current/reference/configuration/framework.html#base-urls) to use the S3 URL as the assets base URL,
-instead of your app domain in production.
+This will deploy a Cloudfront distribution that will act as a proxy: it will serve
+static files directly from S3 and will forward everything else to Lambda. This is very close
+to how traditional web servers like Apache or Nginx work which means you application doesn't need to change!
+For more details, see [the offical documentation](https://github.com/getlift/lift/blob/master/docs/server-side-website.md#how-it-works). 
 
-```yaml
-# config/packages/prod/assets.yaml
+First install the plugin
 
-framework:
-  assets:
-    base_urls: 'https://<bucket-name>.s3.amazonaws.com'
+```bash
+serverless plugin install -n serverless-lift
 ```
 
-If using Webpack Encore, you also need to add the following config at the end of `webpack.config.js`
+Then add this configuration to your `serverless.yml` file.
 
-```js
-// webpack.config.js
-
-if (Encore.isProduction()) {
-    // Note the '/build' at the end of the URL
-    Encore.setPublicPath('https://<bucket-name>.s3.amazonaws.com/build');
-    Encore.setManifestKeyPrefix('build/')
-}
+```yaml
+...
```

Comment:
Maybe we should add a more complete example like in Lift docs ?

Reply:
Yes I think it could be worth it, because so far it could be confusing: "do I keep the lambda functions?" "where does this go in serverless.yml?", etc.

Reply:
I'm actually not sure how to tackle this. I don't want to put the whole serverless.yml either because that would mean having to maintain it in two places (here and the symfony/recipes-contrib repo). And that we tried to avoid in previous PRs like https://github.com/brefphp/bref/pull/923#discussion_r625660759

Maybe something like this would be a good compromise ? Really not sure, any suggestion appreciated :slightly_smiling_face: 

```yaml
service: symfony

provider:
    ...

plugins:
    - ./vendor/bref/bref
    - serverless-lift

constructs:
    website:
        type: server-side-website
        assets:
            '/bundles/*': public/bundles
            '/build/*': public/build
            '/favicon.ico': public/favicon.ico
            '/robots.txt': public/robots.txt
            # add here any file or directory that needs to be served from S3

functions:
   ...
```

Reply:
Yes, this is exactly what I had in mind.

Just to show familiar stuff first, I would put `functions:` above `constructs:`